engine/src/main/java/com/odysseusinc/arachne/executionengine/service/sql/SqlMetadataServiceFactory.java

```diff
@@ -38,6 +38,7 @@ public SqlMetadataService getMetadataService(DataSourceUnsecuredDTO dataSource)
         SqlMetadataService result;
         switch (type) {
             case SNOWFLAKE:
+            case SPARK:
```

Comment:
Why is spark using PostgreSqlMetadataService implementation?

Reply:
@dmitrys-odysseus I would probably switch to `ImpalaMetadataService`